geonode/upload/handlers/common/vector.py

```diff
@@ -414,6 +418,11 @@ def import_resource(self, files: dict, execution_id: str, **kwargs) -> str:
                     )
                     # and layer.GetGeometryColumn() is not None
                 ):
+                    vrt_filename, vrt_layer_name = create_vrt_file(layer, files.get("base_file"))
```

Comment:
@sijandh35 let's keep the generation of the vrt file as optional in the workflo. We can evaluate the attributes, if one of them needs to be modified, we can generate the vrt file, otherwise we can proceed with the normal workflow